2.segmentation/scripts/0.segment_images.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+#
+
+# In[1]:
+
+
+import pathlib
+
+import cv2
+import numpy as np
+
+# from segment_anything import SamPredictor
+import supervision as sv
+import supervision.draw.color as sv_color
+import torch
+from segment_anything import SamAutomaticMaskGenerator, sam_model_registry
+from skimage import io
+from tqdm import tqdm
+
+# ## Define Paths
+
+# In[2]:
+
+
+# import image_path
+image_path = pathlib.Path("../../data/0.raw_images/")
+# max_projection paths
+max_projection_path = pathlib.Path("../../data/1.maximum_projections/")
+# output mask path
+mask_path = pathlib.Path("../../data/2.masks/")
+# sqlite path
+sqlite_path = pathlib.Path("../../data/3.sqlite_output/")
+# models path
+models_path = pathlib.Path("../../data/models/")
+
+# create directories if they don't exist
+max_projection_path.mkdir(parents=True, exist_ok=True)
+mask_path.mkdir(parents=True, exist_ok=True)
+sqlite_path.mkdir(parents=True, exist_ok=True)
+models_path.mkdir(parents=True, exist_ok=True)
+
+
+# In[3]:
+
+
+# Generate a list of all the images in the image_path directory of max projection images
```

Comment:
```suggestion
# Generate a list of all the images in the max_projection_path directory of max projection images
```
Recommend updating this code comment to use the max_projection_path since that is what the code below uses.